app/views/dashboard/locations/new.html.erb

```diff
@@ -1,6 +1,6 @@
 <%= render "dashboard/layouts/main" do %>
   <div class="w-full h-full flex flex-row items-center justify-end">
-    <div class="w-3/5 h-full">
+    <div class="w-4/6 h-full">
```

Comment:
Your math teacher would prefer 2/3

Reply:
Haha true. I tried different things before and then settled for this, didn't rethink it :D